tooling/profiler/src/cli/execution_flamegraph_cmd.rs

```diff
@@ -9,14 +9,15 @@ use nargo::foreign_calls::DefaultForeignCallBuilder;
 use noir_artifact_cli::fs::artifact::read_program_from_file;
 use noir_artifact_cli::fs::inputs::read_inputs_from_file;
 use noirc_artifacts::program::ProgramArtifact;
+use tracing::{debug, info};
 
 use crate::errors::{CliError, report_error};
 use crate::flamegraph::{BrilligExecutionSample, FlamegraphGenerator, InfernoFlamegraphGenerator};
 use crate::opcode_formatter::format_brillig_opcode;
 use bn254_blackbox_solver::Bn254BlackBoxSolver;
 use noirc_artifacts::debug::DebugArtifact;
 
-/// Generates a flamegraph mapping unconstrained Noir execution to source code.
+/// Generates a flamegraph mapping Brillig opcodes to source code.
```

Comment:
This has slightly changed the meaning of this comment. This flamegraph takes into account how many times we touch a particular brillig opcode during execution rather than just mapping brillig opcodes to the source code.